File: php/abc199_a/main.php
<?php

function solve($a, $b, $c)
{
  if ($a * $a + $b * $b < $c * $c) {
    echo "Yes\n";
  } else {
    echo "No\n";
  }
}

function main()
{
  $line = trim(fgets(STDIN));
  [$a, $b, $c] = array_map('intval', explode(' ', $line));
  solve($a, $b, $c);
}

main();

File: Elixir/abc199_a/main.ex
defmodule Main do
  def main do
    [a, b, c] =
      IO.gets("") |> String.trim() |> String.split() |> Enum.map(&String.to_integer/1)

    result = solve(a, b, c)
    IO.puts(result)
  end

  def solve(a, b, c) do
    if a * a + b * b < c * c do
      "Yes"
    else
      "No"
    end
  end
end

Main.main()
